YTIButtonRenderer.h

```diff
@@ -1,4 +1,12 @@
 #import <Foundation/Foundation.h>
+#import "YTICommand.h"
+#import "YTIFormattedString.h"
+#import "YTIIcon.h"
 
 @interface YTIButtonRenderer : NSObject
-@end
\ No newline at end of file
+@property(retain, nonatomic) YTICommand *command;
+@property(retain, nonatomic) YTIIcon *icon;
+@property(retain, nonatomic) YTICommand *navigationEndpoint;
+@property(copy, nonatomic) NSString *targetId; // @dynamic targetId;
```

Comment:
```suggestion
@property (copy, nonatomic) NSString *targetId;
```

Property comments are not needed.